packages/adapters/ethers/src/client.ts

```diff
@@ -309,7 +309,7 @@ export class EthersAdapter extends AdapterBlueprint {
             id: info?.rdns || '',
```

Comment:
Should we consider not adding connector to the list if `name` and `rdns` not valid instead of setting empty values?